packages/host/config/environment.js

```diff
@@ -27,6 +27,7 @@ module.exports = function (environment) {
     matrixURL: process.env.MATRIX_URL || 'http://localhost:8008',
     autoSaveDelayMs: 500,
     monacoDebounceMs: 500,
+    monacoCursorDebounceMs: 200,
```

Comment:
I made new config because `monacoDebounceMs` (500 ms) is too long for initializing cursor position.